src/delta_lake/data_store.py

```diff
@@ -200,3 +200,33 @@ def _get_structure(
         db_structure = _execute_with_spark(_get_structure, spark)
 
     return _format_output(db_structure, return_json)
+
+
+def is_database_exists(
+    database: str,
+    spark: Optional[SparkSession] = None,
+    use_postgres: bool = True,
+) -> bool:
+    """
+    Check if a database exists in the Hive metastore.
+    """
+    return database in get_databases(
+        spark=spark, use_postgres=use_postgres, return_json=False
+    )
```

Comment:
it seems like you ought to be able to just do a specific query for the database rather than listing them all, but there's probably not enough databases that'd this'll be a problem for the near future

Reply:
Same for tables below